src/ints/mouse.cpp

```diff
@@ -31,21 +34,57 @@
 #include "bios.h"
 #include "dos_inc.h"
 
+using namespace bit::literals;
+
+MouseInfoConfig mouse_config;
+MouseInfoVideo mouse_video;
+
+enum class EventType : uint8_t { // compatible with DOS driver mask in driver function 0x0c
+	NotDosEvent    = 0x00,
+	MouseHasMoved  = 0x01,
+	PressedLeft    = 0x02,
+	ReleasedLeft   = 0x04,
+	PressedRight   = 0x08,
+	ReleasedRight  = 0x10,
+	PressedMiddle  = 0x20,
+	ReleasedMiddle = 0x40,
+	WheelHasMoved  = 0x80,
+};
```

Comment:
The hex values are not that intuitive (although yes, eventually it's possible to "see" them as single bits).  Other options:

``` c++
enum class EventType : uint8_t { // compatible with DOS driver mask in driver function 0x0c
	NotDosEvent    = 0b0000'0000,
	MouseHasMoved  = 0b0000'0001,
	PressedLeft    = 0b0000'0010,
	ReleasedLeft   = 0b0000'0100,
	PressedRight   = 0b0000'1000,
	ReleasedRight  = 0b0001'0000,
	PressedMiddle  = 0b0010'0000,
	ReleasedMiddle = 0b0100'0000,
	WheelHasMoved  = 0b1000'0000,
};
```

``` c++
enum class EventType : uint8_t { // compatible with DOS driver mask in driver function 0x0c
	NotDosEvent    = 0 << 0,
	MouseHasMoved  = 1 << 0,
	PressedLeft    = 1 << 1,
	ReleasedLeft   = 1 << 2,
	PressedRight   = 1 << 3,
	ReleasedRight  = 1 << 4,
	PressedMiddle  = 1 << 5,
	ReleasedMiddle = 1 << 6,
	WheelHasMoved  = 1 << 7,
};
```

I prefer the last one, it's very easy to see the shifts incrementing and visually check it.